### ISSUE TITLE
goto definition not works

### ISSUE THREAD

**Issue by flaviolaino** [facebook-atom/atom-ide-ui#302]
### Description
The "goto definition" going to first line of the file instead where function is defined.

### Expected Behavior
going to right line

### Actual Behavior
going to first line of the file where function is defined

### Versions
* Atom: 1.29.0
* Client OS: MacOs 10.13.6
* atom-ide-ui: 0.13.0

### Additional Details
* Installed packages (`apm ls --installed`):
```sh
Community Packages (20) /Users/flavio.laino/.atom/packages
├── atom-autocomplete-php@0.25.6 (disabled)
├── atom-ide-ui@0.13.0
├── atom-symfony2@0.3.0 (disabled)
├── busy-signal@1.4.3
├── custom-window-title@1.0.4
├── goto-definition@1.3.4
├── highlight-selected@0.14.0
├── ide-typescript@0.7.6
├── intentions@1.1.5
├── js-hyperclick@1.13.3 (disabled)
├── language-pug@0.0.22
├── linter@2.2.0
├── linter-php@1.5.1
├── linter-ui-default@1.7.1
├── minimap@4.29.8
├── minimap-git-diff@4.3.1
├── minimap-highlight-selected@4.6.1
├── php-hyperclick@1.2.1 (disabled)
├── php-twig@4.0.0
└── project-manager@3.3.6
```

**Comment by pelmers** [comment]
Hi @flaviolaino, this issue most likely belongs to the ide- package which provides services for the file in question (from the package list, looks like it's https://github.com/atom/ide-typescript).

**Comment by flaviolaino** [comment]
I disabled that package but not resolving.
Here are enabled packages:
```sh
flavio.laino$ apm ls --installed --enabled
Community Packages (15) /Users/flavio.laino/.atom/packages
├── atom-ide-ui@0.13.0
├── busy-signal@1.4.3
├── custom-window-title@1.0.4
├── goto-definition@1.3.4
├── highlight-selected@0.14.0
├── intentions@1.1.5
├── language-pug@0.0.22
├── linter@2.2.0
├── linter-php@1.5.1
├── linter-ui-default@1.7.1
├── minimap@4.29.8
├── minimap-git-diff@4.3.1
├── minimap-highlight-selected@4.6.1
├── php-twig@4.0.0
└── project-manager@3.3.6
```

**Comment by flaviolaino** [comment]
@pelmers issue affects only definitions in the same file of their usage:

```php
<?php

class A{
    public function hi(){
        echo 'hi';
    }

    public function hello(){
        $this->hi(); // clicking on 'hi()' go to line 1
    }
}
```

**Comment by flaviolaino** [comment]
Resolved with Atom 1.30
